### PR TITLE
Feature/iiwa 63

### PR DESCRIPTION
**Pull request by mcdowellalex**
the app redirects to the login page if the context variable givenName is an empty string, which can only not be an empty string after a successful login.

### CODE REVIEW

File: src/frontend-admin/src/components/App/App.tsx [hunk 1]
```diff
@@ -1,71 +1,118 @@
...
+import React, {useState} from 'react'
+import {Route, NavLink, BrowserRouter as Router, Switch, Redirect} from 'react-router-dom'
 
 // Components
 import {DashboardPage} from '../pages/DashboardPage/DashboardPage'
 import {DepartmentsListPage} from '../pages/DepartmentsListPage/DepartmentsListPage'
 import {EmployeesListPage} from '../pages/EmployeesListPage/EmployeesListPage'
 import {HardwareListPage} from '../pages/HardwareListPage/HardwareListPage'
 import {ProgramsListPage} from '../pages/ProgramsListPage/ProgramsListPage'
-import {HelloUser} from '../HelloUser/HelloUser'
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/App/App.tsx]: Was this intended to be removed?

Comment (mcdowellalex) [reply src/frontend-admin/src/components/App/App.tsx]: No

File: src/backend-api/Controllers/LoginController.cs [hunk 2]
```diff
@@ -126,14 +126,25 @@ public IActionResult Login([FromBody] AuthRequest request)
...
+                    returnList.Add(returnInformation);
+                    // return Ok(new
+                    // {
+                    //     Refreshtoken,
+                    //     Accesstoken,
+                    //     accessToken.ValidTo,
+                    //     user.GivenName,
+                    //     isAdmin
+                    // });
+                    return Ok(returnList);
```
Comment (mtleonard120) [inline src/backend-api/Controllers/LoginController.cs]: Mind taking a minute to walk me through your changes here? (I'm predisposed to be suspicious of backend changes on a frontend PR ;) )

Comment (charkour) [reply src/backend-api/Controllers/LoginController.cs]: I helped alex with this. For some reason if the data is not returned as an array, the data in the axios response is an empty string. Don't know why axios doesn't like data as an object. It doesn't seem to be a common issue.

Comment (mtleonard120) [reply src/backend-api/Controllers/LoginController.cs]: Thanks for the explanation - that's weird :/

**Pull request merged**